elements/div.html

```diff
@@ -0,0 +1,51 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        &lt;div&gt;
+      </code>
+    </h2>
+    <div class="base-words">
+      <div class="base-word-line">
+        English Base: <strong>Divider</strong>
+      </div>
+      <div class="base-word-line">
+        Spanish translation/Traducción en Español: <strong></strong>
+      </div>
+    </div>
+  </header>
+  <div class="definition">
+    <div class="definition-box definition-spanish">
+      <h4>Definici&oacute;n</h4>
+      <p></p>
+    </div>
+    <div class="definition-box definition-english">
+      <h4>Definition</h4>
+      <p>An all purpose, generic container for web content.  There are more specific containers (header, footer, nav, section, etc) for more specific use cases.  The div container is used when none of these apply.  If you're grouping content for generic purposes like styling or simply logical grouping, use a div.</p>
+    </div>
+  </div>
+  <div class="how-to-remember">
+    <div class="how-to-remember-box how-to-remember-spanish">
+      <h4>&iquest;C&oacute;mo recordar?</h4>
+      <p></p>
+    </div>
+    <div class="how-to-remember-box how-to-remember-english">
+      <h4>How to Remember</h4>
+      <p>Div is used to divide your web page into similarly styled sections.</p>
+    </div>
+  </div>
+  <pre class="code-example">
+    <code>
+      <strong>&lt;div class="chapter-list"&gt;</strong>
+          &lt;ul&gt;
```

Comment:
the indentation in this code example is excessive. make sure your editor is setup to be 2 spaces for a tab, instead of 4. i realize it depends on the preference of the developer, but 2 space indentation is what's being used in the rest of the site.